### ISSUE TITLE
Rename to ignore-require?

### ISSUE THREAD

**Issue by MoOx** [bkonkle/ignore-styles#10]
Since this module is more a way to ignore any file extensions, maybe a rename can be a good idea :)

**Comment by bkonkle** [comment]
Yep, I've been contemplating a name change for a little while now. Your suggestion is perfect! When I get some more time, I'll look into the best way to make the switch. 👍

**Comment by MoOx** [comment]
You will have to:
- deprecate ignore-style on npm via `$ npm deprecate ignore-style@"< 4.0.0" "ignore-style is now ignore-require"` (use the latest version)
- rename repo on github
- rename package in package.json
- update repo url in package.json
- npm publish (with new name) :)

**Comment by iamstarkov** [comment]
ignore-require seems like a proper name for this
